vendor/engines/secure_rooms/app/models/secure_rooms/card_reader.rb

```diff
@@ -0,0 +1,9 @@
+module SecureRooms
+
+  class CardReader < ActiveRecord::Base
+
+    belongs_to :secure_room, foreign_key: :product_id
```

Comment:
yeah, this PR is primarily just to spit out the "deny" so these are only here for their finders to fail